### PR TITLE
Copy edit player-tags-create.md

### PR DESCRIPTION
**Pull request by amanda-butler-unity**
Copy edit file for typos and style.

#### **WARNING**: Please make your PR against the `develop` branch.
- - -
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

**Issue Number:**

### CODE REVIEW

File: mppm/player-tags/player-tags-create.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ title: Create a Player Tag
 description: Use these steps to create Player Tags to configure the player actions in Multiplayer Play Mode.
 ---
 
-To create a **Player Tag**, perform mthe following actions: 
+To create a **Player Tag**, perform the following actions: 
 
-1. In the **Unity Editor**, open the **MPPM** window by navigating to **Window** > **Multiplayer Play Mode**.
-2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create your new tag.
+1. In the **Unity Editor**, open the **Multiplayer Play Mode** window by navigating to **Window** > **Multiplayer Play Mode**.
+2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create a new tag.
```
Comment (amanda-butler-unity) [inline mppm/player-tags/player-tags-create.md:10]:
```suggestion
1. Select **Add Tags for Player**.
1. In the dropdown list, select **+ Create Tag**
```

File: mppm/player-tags/player-tags-create.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ title: Create a Player Tag
...
 3. Name the **Player Tag**.
-4. Select **Save**. Your new tag is automatically added to the `VirtualProjectsConfig.json` file in the `ProjectsSettings` folder.
-5. Your new tag should also appear under the **Player Tags** section of the **MPPM** window and in the dropdown menu for the **Tag** option of each **Player**.
+4. Select **Save**. The new tag is automatically added to the `VirtualProjectsConfig.json` file in the `ProjectsSettings` folder.
```
Comment (amanda-butler-unity) [inline mppm/player-tags/player-tags-create.md:12]: Where is this folder? I was going to check that there's an "s" on the end of "Projects," but I don't know where to find the folder.

File: mppm/player-tags/player-tags-create.md (hunk 1 above)
Comment (amanda-butler-unity) [reply mppm/player-tags/player-tags-create.md:10]: Does Docusaurus accept the "1. 1. 1." numbering like Github-flavored Markdown, or do we need to update all of the numbers here?

Comment (amanda-butler-unity) [reply mppm/player-tags/player-tags-create.md:10]: In fact, we may want to replace this with the wording from https://docs-multiplayer.unity3d.com/mppm/current/player-tags/player-tags-attach/ for consistency.

File: mppm/player-tags/player-tags-create.md (hunk 2 above)
Comment (Vic-Cooper) [reply mppm/player-tags/player-tags-create.md]: Following this one up with Johann

File: mppm/player-tags/player-tags-create.md (hunk 1 above)
Comment (Vic-Cooper) [reply mppm/player-tags/player-tags-create.md]: Updated list and instructions for consistency, thanks :)

File: mppm/player-tags/player-tags-create.md (hunk 2 above)
Comment (Vic-Cooper) [reply mppm/player-tags/player-tags-create.md]: This is in the project directory, I'll update the instructions to state this :)
image

File: mppm/player-tags/player-tags-create.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-To create a **Player Tag**,
-
-1. In the **Unity Editor**, open the **MPPM** window by navigating to **Window** > **Multiplayer Play Mode**.
-2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create your new tag.
-3. Name the **Player Tag**.
-4. Select **Save**. Your new tag is automatically added to the `VirtualProjectsConfig.json` file in the `ProjectsSettings` folder.
-5. Your new tag should also appear under the **Player Tags** section of the **MPPM** window and in the dropdown menu for the **Tag** option of each **Player**.
+To create a **Player Tag**, perform the following actions:
 
+1. Open the Multiplayer Play mode window (**Window** > **Multiplayer Play Mode**).
```
Comment (amanda-butler-unity) [inline mppm/player-tags/player-tags-create.md:10]:
```suggestion
1. To open the Multiplayer Play mode window, navigate to **Window** > **Multiplayer Play Mode**.
```

**Pull request merged**